dlt/cli/_dlt.py

```diff
@@ -188,8 +194,8 @@ def main() -> int:
 
     deploy_cmd = subparsers.add_parser("deploy", help="Creates a deployment package for a selected pipeline script")
     deploy_cmd.add_argument("pipeline_script_path", metavar="pipeline-script-path", help="Path to a pipeline script")
-    deploy_cmd.add_argument("deployment_method", metavar="deployment-method", choices=["github-action"], default="github-action", help="Deployment method")
-    deploy_cmd.add_argument("--schedule", required=True, help="A schedule with which to run the pipeline, in cron format. Example: '*/30 * * * *' will run the pipeline every 30 minutes.")
+    deploy_cmd.add_argument("deployment_method", metavar="deployment-method", choices=[DeploymentMethods.github_actions.value, DeploymentMethods.airflow.value], default=DeploymentMethods.github_actions.value, help="Deployment method")
+    deploy_cmd.add_argument("--schedule", required=False, help="A schedule with which to run the pipeline, in cron format. Example: '*/30 * * * *' will run the pipeline every 30 minutes.")
```

Comment:
In case of Airflow I do not use schedule, because I do not change the dag script. I thought to create some yaml with dag configurations, but I am not sure it is necessary. what do you think?

Reply:
isn't the schedule part of the @dag definition? but then we need to modify the deployment script. we could also use `config.toml` for that. just leave this open. we need to merge a first version :)

Reply:
> isn't the schedule part of the @dag definition?
yeah, but I decided do not change python script with dag definition for a while... we can think and change it bit later